docs/docs/self-hosting/index.md

```diff
@@ -16,7 +16,25 @@ the same code we use for our own cloud service.
 
 ## Getting started
 
-Start the server
+### Dependencies 
+
+Please follow the steps below to install all the necessary dependencies to run Ente on a fresh server or VPS.
+
+The commands and package names may vary depending on the operating system and it's package managers.
+
+```sh 
+# installing npm and yarn 
+
+sudo apt update 
+sudo apt install nodejs npm
+sudo npm install -g yarn // to install yarn globally
+``` 
```

Comment:
Just to be clear, these are not required for self-hosting ente-server. These are only required if you are planning to build the webapp/desktop app. We can probably mention this under hosting web app section

Reply:
Ok, I'll make the changes. 

Thank You

Reply:
did the necessary changes.